assets/questions/bomberman.js

```diff
@@ -0,0 +1,146 @@
+                else:
+                    enemyKilled[r][c] += right + down[c]
+                    if enemyKilled[r][c] > maxEnemyKilled:
+                        maxEnemyKiled = enemyKilled[r][c]
+        return maxEnemyKilled
+
+    @classmethod
+    def bombAllEnemiesOnBoard(cls, board):
+        if not board or not board[0]: return 0
+        count = 0
+        for r in range(len(board)):
+            for c in range(len(board[0])):
+                if board[r][c] == "e":
+                    count += 1
+        return count
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'For this question, you will be given a board, in which all elements are strings. ',
+        'x represents a wall, e represents an enemy, "" represents an empty cell. ',
+        'You can place an bomb in an empty cell, which will kill all enemies in vertical and horizontal ',
+        'direction until a wall or board boundary is met. ',
+        'Please write a function bomb(board), to find the best position to place a bomb in order to',
+        'kill as many enemies as possible, then return the number of enemies killed.'
+      ].join('')
+    ],
+    prerequisiteSkills: ['Arrays', 'Strings', 'String Manipulation'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'bomb',
+    correctnessTests: [
+      {
+        input: [],
+        allowedOutputs: [0]
+      },
+      {
+        input: [[], [], []],
+        allowedOutputs: [0]
+      },   
+      {
+        input: [["e", "", "e", "x", "", "e"]],
+        allowedOutputs: [2]
+      },
+      {
+        input: [["e", "", "e", "", "", "e"]],
+        allowedOutputs: [3]
+      },
+      {
+        input: [["e"], ["x"], [""], ["x"]],
+        allowedOutputs: [0]
+      },
+      {
+        input: [["e"], ["e"], [""], ["x"]],
+        allowedOutputs: [2]
+      },
+      {
+        input: [["", "e", ""], ["e", "", "e"], ["", "e", ""]],
+        allowedOutputs: [4]
+      }
+    ],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.ignoreWalls',
+      messages: [["Try to run your code on ['e', '', 'x', 'e']. ",
+        "Is the result match what you expected? "
+      ].join(''),
+        [
+          "You seem to be ignoring walls in your solution, which is incorrect.",
+          "If a bomb is placed, the resulting explosion should only go until it reaches a wall."
+        ].join('')
+      ]
+    },
+    {
+      buggyFunctionName: 'AuxiliaryCode.bombAllEnemiesOnBoard',
+      messages: [["Try to run your code on ['e', 'x', '', 'x', 'e']. ",
+        "Is the result match what you expected?"
+      ].join(''),
+        [
+          "You seemed to be returning all the enemies on the board. ",
+          "Your bomb can only kill enemies in the same row and column. "
+        ].join('')
+      ]
+    }],
+    // No performance test here. Since the desired runtime complexity is O(N^2),
```

Comment:
// Since we currently only support O(n) runtime complexities and this requires an O(n^2) solution, we have skipped performance tests.

You should also file an issue + a TODO.